src/functions.php

```diff
@@ -2,14 +2,14 @@
 
 namespace Amp\ParallelFunctions;
 
+use function Amp\call;
 use Amp\MultiReasonException;
 use Amp\Parallel\Sync\SerializationException;
+use function Amp\Parallel\Worker\enqueue;
 use Amp\Parallel\Worker\Pool;
 use Amp\Promise;
-use Opis\Closure\SerializableClosure;
-use function Amp\call;
-use function Amp\Parallel\Worker\enqueue;
 use function Amp\Promise\any;
+use Laravel\SerializableClosure\SerializableClosure;
```

Comment:
The imports are misordered now, probably an issue with the new php-cs-fixer?

Reply:
Missed that it's in the other PR, so should probably be fixed after rebase + running the fixer?